pos_stock_available_online/models/pos_session.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models
+
+
+class PosSession(models.Model):
+    _inherit = "pos.session"
+
+    def _process_pos_ui_product_product(self, products):
+        config = self.config_id
+        if config.display_product_quantity:
```

Comment:
return early and save a identation level
```suggestion
        if not config.display_product_quantity:
            return super()._process_pos_ui_product_product(products)
```

Reply:
Hello, @hugho-ad the negative condition is more difficult to read code I see no reason to change in this way

Reply:
not it's not, is called early return pattern
https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
@hugho-ad we have to return "super" in any case. In your case, you offer to return only if this setting is not specified, which is not true.